Packs/CommonScripts/Scripts/EvaluateContextValue/EvaluateContextValue_test.py

```diff
@@ -0,0 +1,83 @@
+import re
+import json
+import fnmatch
+import pytest
+from typing import Any
+import EvaluateContextValue
+import demistomock as demisto
+
+
+def equals_object(obj1, obj2) -> bool:
+    if type(obj1) is not type(obj2):
+        return False
+    elif isinstance(obj1, dict):
+        for k1, v1 in obj1.items():
+            if k1 not in obj2:
+                return False
+            if not equals_object(v1, obj2[k1]):
+                return False
+        return not (set(obj1.keys()) ^ set(obj2.keys()))
+    elif isinstance(obj1, list):
+        # Compare lists (ignore order)
+        list2 = list(obj2)
+        for _i1, v1 in enumerate(obj1):

```

Comment:
The loop variable `_i1` is not used. Please replace it with an underscore (`_`) to signify that it is intentionally ignored.